src/components/include/application_manager/application_manager.h

```diff
@@ -393,15 +393,32 @@ class ApplicationManager {
    *
    * @return true on success, false if passthru is already in process
    */
+  // DEPRECATED
   virtual bool BeginAudioPassThrough() = 0;
```

Comment:
In my experience, the `DEPRECATED` macro occasionally causes issues if it is used in both the header and implementation. The best solution I've found is just to use the macro in the highest level (which would be this interface in this case). In any case, using this setup it shouldn't be present in any `.cc` files (which I will need to fix in the current develop).

Reply:
Thanks for your comment. I reverted the last change and put DEPRECATED macro only in application_manager.h. I also put it in the mock file.